habitat-baselines/habitat_baselines/rl/multi_agent/multi_agent_access_mgr.py

```diff
@@ -234,12 +234,23 @@ def _sample_active_idxs(self) -> Tuple[np.ndarray, np.ndarray]:
                 raise ValueError(
                     "The current code only supports sampling one agent of a given type at a time"
                 )
-            active_agents_type = self._rnd.choice(
-                self._agent_count_idxs[agent_type_ind],
-                size=self._pop_config.num_active_agents_per_type[
-                    agent_type_ind
-                ],
-            )
+            if (
+                self._pop_config.force_partner_sample_idx >= 0
```

Comment:
flagging that we call _pop_config many times in this piece of code. Might be better to 
`force_partner_sample_idx = self._pop_config.force_partner_sample_idx ` at the very start.